### ISSUE TITLE
Promote aurelia-typed-observable plugin to official and add docs

### ISSUE THREAD

**Issue by zewa666** [aurelia/documentation#411]
Just today I again stumbled across this great plugin https://github.com/aurelia-contrib/aurelia-typed-observable-plugin which definitely deserves more visibility. How about promoting it to an official one and adding some docs @EisenbergEffect, @bigopon ?

**Comment by EisenbergEffect** [comment]
@bigopon How much code is it? This is something I'd be interested in having in the core for vNext actually, but we could keep the plugin for vCurrent. Thoughts?

**Comment by bigopon** [comment]
For vNext, its simple to add, and should in core.
Back to the Q, for vCurrent, how do we promote the plugin to an official one?

**Comment by zewa666** [comment]
@eisenbergeffect creates a new repo, gives you write permissions and off you go with the Initial commit.

I also like the idea of having this in core for vNext but so far the Plugin approach is good. Essentially what it really boils down to is having an official documentation and the proof for consumers that it will be maintained

**Comment by EisenbergEffect** [comment]
Yep, @bigopon I could set up a repo for you under the `aurelia` org and then we just need to add a doc to the plugins section of our site. You want to go for it?

**Comment by bigopon** [comment]
@EisenbergEffect the plugins require devs to import decorators from there, not from core module. Does it look good for an official plugin?

**Comment by EisenbergEffect** [comment]
I think it's something we can live with for now as we transition to vNext.

**Comment by zewa666** [comment]
I'd agree. Importing from other Plugin packages shouldnt be an issue If we clearly document this, maybe even with a hint that it wont be necessary in vNext

**Comment by bigopon** [comment]
Changing from its current repo to aurelia org is easy. What do we want to do with release on npm? And potentially pagkage name as well?

**Comment by 3cp** [comment]
npm has command to remove all existing versions.

But you need to wait 24 hrs (some npmjs policy) before republish it under different npm user name. For safety, you'd better republish all versions.

**Comment by bigopon** [comment]
@EisenbergEffect @zewa666 any comments on what @3cp and @bigopon said?

**Comment by EisenbergEffect** [comment]
For the NPM release, you just need to make AureliaEffect an owner. I don't think there's a need to change the name or republish stuff.

**Comment by michaelw85** [comment]
I would very much like to see this made official or even better moved into the core (not just vnext). I've looked for type coercion in the past and today I was looking for default binding mode for view only components and stumbled upon https://github.com/aurelia/templating-resources/pull/309

Can I pitch in to make this happen?

**Comment by bigopon** [comment]
@michaelw85 that'd be awesome. But the first thing we need to sort out is the scope of the plugin. At the moment, it's for coercing value, we can expand it to provide a few extra bits of functionalities

@EisenbergEffect

**Comment by EisenbergEffect** [comment]
I'd like to get the plugin made official as is first. Then, I'd like to build this and the other issue into vNext. Once that's done, we should look at what the best way to realize this feature is in vCurrent. This will ensure that vCurrent is at least a strict subset of vNext and help us make them compatible.

@bigopon I know Fred is in the middle of a big refactor on vNext. Maybe check with him on this. If this work is mostly restricted, at least at first, to the template compiler, then you might be able to work on it in parallel.
